src/components/OptionsPicker/OptionItem.tsx

```diff
@@ -54,10 +54,7 @@ function OptionItem({title, icon, onPress, isSelected = false, isDisabled, style
                         />
                         {!isDisabled && (
                             <View>
-                                <SelectCircle
-                                    isChecked={isSelected}
-                                    selectCircleStyles={styles.sectionSelectCircle}
-                                />
+                                <SelectCircle isChecked={isSelected} />
```

Comment:
Have you checked if it looks like it should on both themes? E.g like this for dark theme 

![Screenshot 2024-06-06 at 14 45 57](https://github.com/Expensify/App/assets/44364829/4950c49e-c8d2-47a1-b1cc-135a3da9fe53)

I haven't ran your code but I made that change on my branch and the background is too dark

Reply:
I wasn't aware that it's too dark now. I've reverted it to default state that should work for both platforms. Changing it again so its fine on both themes